configs/nx.json

```diff
@@ -23,22 +23,20 @@
   "sitemap_alternate_links": true,
   "selectors": {
     "lvl0": {
-      "selector": "#nav h5",
+      "selector": "//nav[contains(@id, 'nav')]//span[contains(@class, 'text-gray-900')][1]/preceding::h5[1]",
```

Comment:
Hey, thanks for the PR.

Due to future changes in our crawler, we recommend users to find a DOM alternative to xPaths selectors.

I'm still merging it for now since it will still work

Reply:
Thanks for the heads up!